data/assets/scene/solarsystem/missions/insight/edl.asset

```diff
@@ -151,7 +151,7 @@ local RotationKeyframes = {
 
 local InsightParent = {
   Identifier = "InsightParent",
-  Parent = mars.Identifier,
+  Parent = mars.Mars.Identifier,
```

Comment:
Very strange that that didn't throw a Lua error;  with the addition of the strict-mode it should have thrown a Lua error when trying to access a non-existing table field